### PR TITLE
a9-float-layout

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

File: src/mm/html-css/a9-float-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Iregui44) [inline src/mm/html-css/a9-float-layout/index.html:5]:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

File: src/mm/html-css/a9-float-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <title>Float Layout</title>
```
Comment (Iregui44) [inline src/mm/html-css/a9-float-layout/index.html:6]: You can add a favicon for better styles in the explorer :)

File: src/mm/html-css/a9-float-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="sidebar">
+      <div class="profile">
+        <img
+          src="https://scontent.fbog10-1.fna.fbcdn.net/v/t39.30808-6/378116736_2044295869255074_5106250695232773874_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=6ee11a&_nc_eui2=AeHOJa0BBNu9Eqzj6rRiRGnHbzFk9p0Zb5NvMWT2nRlvk1T1zaIAj3ahB4JCCpuLqihkZTu4jW-txarDepfiJMe8&_nc_ohc=Y7yAo4qyHLAQ7kNvgGCO19B&_nc_ht=scontent.fbog10-1.fna&oh=00_AYB3FK-AdC9V_mVswMWLq5e-As5pyf68oqagm9bii2qc-g&oe=66A8A8F7"
```
Comment (Iregui44) [inline src/mm/html-css/a9-float-layout/index.html:17]: alt attribute

File: src/mm/html-css/a9-float-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
+body {
+  background-image: url(https://images.unsplash.com/photo-1645536908932-652fbd998029?q=80&w=2000&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D);
+  font-family: Arial, sans-serif;
+  background-color: #000;
+  color: #fff;
+  margin: 0%;
```
Comment (Iregui44) [inline src/mm/html-css/a9-float-layout/styles.css:6]: I think you can use the 0 alone why the %?

File: src/mm/html-css/a9-float-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  padding: 0%;
+}
+.sidebar {
```
Comment (Iregui44) [inline src/mm/html-css/a9-float-layout/styles.css:9]: Space

File: src/mm/html-css/a9-float-layout/index.html (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/html-css/a9-float-layout/index.html:5]: When generating the commit the "/" is automatically placed

File: src/mm/html-css/a9-float-layout/index.html (hunk 2 above)
Comment (valentinavargas31) [reply src/mm/html-css/a9-float-layout/index.html:7]: ok

File: src/mm/html-css/a9-float-layout/index.html (hunk 3 above)
Comment (valentinavargas31) [reply src/mm/html-css/a9-float-layout/index.html]: corrected

File: src/mm/html-css/a9-float-layout/styles.css (hunk 4 above)
Comment (valentinavargas31) [reply src/mm/html-css/a9-float-layout/styles.css]: I thought it was mandatory to put it jeje

File: src/mm/html-css/a9-float-layout/styles.css (hunk 5 above)
Comment (valentinavargas31) [reply src/mm/html-css/a9-float-layout/styles.css:10]: ok

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**